apps/web-giddh/src/app/shared/header/components/group-update/group-update.component.ts

```diff
@@ -191,8 +190,7 @@ export class GroupUpdateComponent implements OnInit, OnDestroy, AfterViewInit {
                 let arr: IOption[] = [];
                 if (taxes) {
                     if (activeGroup) {
-                        let applicableTaxes = activeGroup.applicableTaxes.map(p => p.uniqueName);
-
+                        let applicableTaxes = activeGroupTaxHierarchy.applicableTaxes.map(p => p.uniqueName);
```

Comment:
add nulity check for this 
```suggestion
                        let applicableTaxes = activeGroupTaxHierarchy?.applicableTaxes.map(p => p.uniqueName);
```